mozetl/shield/containers.py

```diff
@@ -0,0 +1,34 @@
+from pyspark.sql.types import StringType, LongType
+
+from ..basic import convert_pings, DataFrameConfig
+from .utils import testpilot_etl_boilerplate
+
+
+def transform_pings(sqlContext, pings):
+    return convert_pings(
+        sqlContext,
+        pings,
+        DataFrameConfig([
+            ("uuid", "payload/payload/uuid", None, StringType()),
+            ("userContextId", "payload/payload/userContextId", None, StringType()),
+            ("clickedContainerTabCount",
+             "payload/payload/clickedContainerTabCount", None, LongType()),
+            ("eventSource", "payload/payload/eventSource", None, StringType()),
+            ("event", "payload/payload/event", None, StringType()),
+            ("hiddenContainersCount", "payload/payload/hiddenContainersCount", None, LongType()),
+            ("shownContainersCount", "payload/payload/shownContainersCount", None, LongType()),
+            ("totalContainersCount", "payload/payload/totalContainersCount", None, LongType()),
+            ("totalContainerTabsCount",
+             "payload/payload/totalContainerTabsCount", None, LongType()),
+            ("totalNonContainerTabsCount",
+             "payload/payload/totalNonContainerTabsCount", None, LongType()),
+            ("pageRequestCount", "payload/payload/pageRequestCount", None, LongType()),
+            ("test", "payload/test", None, StringType()),
+        ], lambda ping: ping['payload/test'] in ["@testpilot-containers", "@shield-study-containers"])
+    )
+
+
+etl_job = testpilot_etl_boilerplate(
+    transform_pings,
+    's3n://telemetry-parquet/harter/containers_testpilottest/v2'
```

Comment:
You should probably choose a new location for this table. Storing this table as v2 will cause [parquet2hive](https://github.com/mozilla/parquet2hive) to load the new shield table into containers_testpilottest, so only the combined shield/txp table will be available from redash. 

Maybe `s3n://telemetry-parquet/harter/containers_txp_shield/v1` ?